src/networkParams.ts

```diff
@@ -63,31 +59,57 @@ export class TransactionOptions {
     }
 
     /**
-     * Creates a TransactionOptions object with the default options setting
+     * Creates a TransactionOptions object with the default options.
      */
-    static withDefaultOptions(): TransactionOptions {
+    static withDefaultOptions() {
         return new TransactionOptions(TRANSACTION_OPTIONS_DEFAULT);
     }
 
     /**
-     * Creates a TransactionsOptions object with the setting for hash signing
+     * Creates a TransactionOptions object from a set of options.
+     */
+    public static withOptions(options: {
```

Comment:
Or should we name it `fromFlags()`, `withFlags()`?

Reply:
My opinion is that we should stay with "withOptions( )" given the  `TransactionOptions` name.
Does it make sense to keep the function `withOptions( )` and remove the "with" prefix from the options names?

Reply:
Indeed. Fixed.